ibis/expr/operations.py

```diff
@@ -2363,6 +2367,11 @@ class ExtractMillisecond(ExtractTimestampField):
     pass
 
 
+class Time(TemporalUnaryOp):
```

Comment:
I think this should inherit from `UnaryOp` since if you inherit from `TemporalUnaryOp` then `date`-typed columns will inherit the `time()` method and that doesn't make sense.